src/spec.c

```diff
@@ -82,7 +78,11 @@ static int zlog_spec_write_time_internal(zlog_spec_t * a_spec, zlog_thread_t * a
 
 	/* When this event's last cached time_local is not now */
 	if (*time_sec != now_sec) {
-		time_stamp_convert_function(&(now_sec), time);
+		if (use_utc) {
+			gmtime_r(&(now_sec), time);
+		} else {
```

Comment:
On MinGW gmtime_r and localtime_r are inline functions, can't get function pointers